host/lottery.html

```diff
@@ -99,33 +104,48 @@
 //Definiowanie zmiennych, sprawdzamy czy w pamięci jest zapisana lista graczy z poprzedniego losowania
 let listaGraczy = [];
 let storedPamiec = JSON.parse(localStorage.getItem("losowaniePamiec"));
+let gracz = "";
+
+//Tutaj będzie pobieranie pliku z listą graczy z serwera
+async function getNames () {
+    const response = await fetch('./names.json');
+    const importLista = await response.json();
+    listaGraczy = importLista;
+    displayLosowanie = listaGraczy.join('\n');
+    document.getElementById("lista").value = displayLosowanie;
+    document.getElementById("pozostalo").value = "Pozostało " + importLista.length + " graczy";
+    document.getElementById("losowany").style.backgroundColor="#7a1414"
+    gracz = "Import poprawny - liczba graczy: " + importLista.length;
```

Comment:
This prints on red background, which is confusing, since everything went OK. Please change the color to something more natural.

Reply:
Changed to sky blue in 1.4.1